fastify.js

```diff
@@ -88,8 +88,13 @@ function fastify (options) {
 
   validateBodyLimitOption(options.bodyLimit)
 
+  const caseSensitiveParse = (string) => {
+    const shouldLower = options.caseSensitive === false
+    return querystring.parse(shouldLower ? string.toLowerCase() : string)
+  }
+
   const requestIdHeader = options.requestIdHeader || defaultInitOptions.requestIdHeader
-  const querystringParser = options.querystringParser || querystring.parse
+  const querystringParser = options.querystringParser || caseSensitiveParse
```

Comment:
Please move the from within the function to the initialization of fastify.

Reply:
Where is that? I tried to move it outside of the `fastify` function but I need access to `options` to know when to lowercase